app/controllers/scans_controller.rb

```diff
@@ -27,20 +27,11 @@ def new_request_params
   end
 
   def redirect_to_success_with_token
-    redirect_to illiad_query(current_request)
+    redirect_to illiad_url
```

Comment:
The user's sunset id does get passed along in the request header to match the user to their illiad account. The illiad.dll file is the 'controller' for that. As long as we can still mimic that on the back end I suppose it could be possible.